### PR TITLE
Change wrong words korean README

### PR DESCRIPTION
**Pull request by techinpark**
## Description

 I naturally changed the awkward parts.
Thanks for pretty projects. 👍

### CODE REVIEW

File: docs/README.kr.md [hunk 1]
```diff
@@ -11,17 +11,33 @@
 <br></br>
 [![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/dawnlabs/carbon&env=TWITTER_CONSUMER_KEY&env=TWITTER_CONSUMER_SECRET&env=TWITTER_ACCESS_TOKEN_KEY&env=TWITTER_ACCESS_TOKEN_SECRET&env=LOGS_SECRET_PREFIX)
 
+
```
Comment (mfix22) [inline docs/README.kr.md]: Please don't add the "Translations" section here. I want to isolate the list to a single spot.

**Comment by mfix22** [comment]
@wooooooak can you take a look at these changes and approve, or leave comments otherwise?

File: docs/README.kr.md (hunk 1 above)
Comment (techinpark) [reply docs/README.kr.md]: @mfix22 Okay. Thank you for your quick response. 😄

**Comment by wooooooak** [comment]
@techinpark @mfix22 cooool. Thanks for better idea~!

File: docs/README.kr.md [hunk 2]
```diff
@@ -11,17 +11,33 @@
...
+##### Translations
```
Comment (mfix22) [inline docs/README.kr.md]: I just meant can you remove these line: https://github.com/dawnlabs/carbon/pull/763/files/42131c69d2c87a876def06d96be53965295df3bc#diff-8c0a439f8e009dcb2fdd6b1aca936db9R15

Lines 15 though 29. There should be no "##### Translations" section in this file

Comment (techinpark) [reply docs/README.kr.md]: @mfix22 yes. i miss understand words. thanks :)

File: docs/README.kr.md [hunk 3]
```diff
@@ -11,22 +11,6 @@
 <br></br>
 [![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/dawnlabs/carbon&env=TWITTER_CONSUMER_KEY&env=TWITTER_CONSUMER_SECRET&env=TWITTER_ACCESS_TOKEN_KEY&env=TWITTER_ACCESS_TOKEN_SECRET&env=LOGS_SECRET_PREFIX)
 
-
-##### Translations
-
```
Comment (techinpark) [inline docs/README.kr.md]: @mfix22 it's correct?

Comment (mfix22) [reply docs/README.kr.md]: 👍

**Comment by mfix22** [comment]
@wooooooak this looks good to you? 👍 👎? 🙂

**Comment by techinpark** [comment]
@mfix22 thanks for merge! 
can you add me `README.md` in Contributors section?

**Pull request merged**